web/src/modules/layout/components/Header.js

```diff
@@ -100,41 +100,40 @@ const NavigationBar: StyledComponent<{||}, ThemeType, *> = styled.div`
  * of the Header.
  * Uses Headroom to save space when scrolling.
  */
-export class Header extends React.PureComponent<PropsType> {
-  static defaultProps = {
-    navigationItems: [],
-    actionItems: []
-  }
-
-  render () {
-    const { viewportSmall, onStickyTopChanged, actionItems, logoHref, navigationItems, platform, cityName } = this.props
-    const { headerHeightSmall, headerHeightLarge } = dimensions
-    const hasNavigationBar = navigationItems.length > 0
-    const height = viewportSmall
-      ? (1 + (hasNavigationBar ? 1 : 0)) * headerHeightSmall + (cityName ? HEADER_TITLE_HEIGHT : 0)
-      : (1 + (hasNavigationBar ? 1 : 0)) * headerHeightLarge
-    const scrollHeight = viewportSmall
-      ? headerHeightSmall + (cityName ? HEADER_TITLE_HEIGHT : 0)
-      : headerHeightLarge
-    return (
+export const Header = ({
+  viewportSmall,
+  onStickyTopChanged,
+  actionItems = [],
+  logoHref,
+  navigationItems = [],
+  platform,
+  cityName
+}: PropsType) => {
+  const { headerHeightSmall, headerHeightLarge } = dimensions
+  const hasNavigationBar = navigationItems.length > 0
+  const height = viewportSmall
+    ? (1 + (hasNavigationBar ? 1 : 0)) * headerHeightSmall + (cityName ? HEADER_TITLE_HEIGHT : 0)
+    : (1 + (hasNavigationBar ? 1 : 0)) * headerHeightLarge
+  const scrollHeight = viewportSmall
+    ? headerHeightSmall + (cityName ? HEADER_TITLE_HEIGHT : 0)
+    : headerHeightLarge
+
+  return (
       <Headroom onStickyTopChanged={onStickyTopChanged}
                 scrollHeight={scrollHeight}
                 height={height}
                 positionStickyDisabled={platform.positionStickyDisabled}>
         <HeaderContainer>
           <Row hasTitle={!!cityName}>
-            <HeaderLogo link={logoHref}
-                        src={buildConfig().icons.appLogo}
-                        alt={buildConfig().appName} />
+            <HeaderLogo link={logoHref} buildConfig={buildConfig()} />
```

Comment:
Imo it is better to call `buildConig()` more locally, as it prevents those blown up snapshots. Is there a reason to pass it as property?

Reply:
Testing was easier that way. Do you think it is better not to pass the build config as property apart from those blown up snaps? Those will be removed anyway sooner or later, so I figured it does not really matter.

Reply:
If it simplifies testing then I guess it's fine. As stated, it's just my personal preference.